docs/_data/components/schemas/data-provenance.yaml

```diff
@@ -0,0 +1,10 @@
+type: object
+properties:
+  provider:
+    $ref: ./entity-id.yaml
+  created-time:
+    $ref: ./timestamp.yaml
+  origin-id:
+    $ref: ./generic-identifier.yaml
+required:
+  - provider
```

Comment:
ok, to require other fields, include more documentation on the intention/purpose of the origin-id in particular.